docs/tutorials/customSessionKeys/createSession.md

```diff
@@ -0,0 +1,253 @@
+---
+sidebar_label: 'Session Module'
+sidebar_position: 5
+---
+# Session Module
+
+In this guide, we will walk through creating a basic Node.js script using
+TypeScript that allows user to create a session.
+
+:::info This tutorial has a previous other steps in the previous sections:
+[Environment Setup](environmentsetup) and
+[Initializing Account](initializeaccount)
+ :::
+
+This tutorial will be done on the Polygon Mumbai Network. We will be using session Module for this.
+
+We will import sessionKeyManagerModule and DEFAULT_SESSION_KEY_MANAGER_MODULE from Biconomy Modules package.
+
+First of all we will initialise the sessionFileStorage using our custom File storage
+
+```typescript
+const sessionFileStorage: SessionFileStorage = new SessionFileStorage(address)
```

Comment:
maybe describe the motivation behind SessionFileStorage and how it implements ISessionStorage. in the beginning you can say this example will create a custom storage client which maintains leaves information in a file but one can have their own implementation and store it in their managed database server